android/src/main/AndroidManifest.xml

```diff
@@ -19,5 +20,9 @@
       <meta-data
         android:name="com.google.android.gms.ads.flag.OPTIMIZE_AD_LOADING"
         android:value="${appJSONGoogleMobileAdsOptimizeAdLoading}"/>
+      <property
```

Comment:
I have seen these warnings before and personally tried to track them down.
I've learned they are not avoidable, but people (like myself!) will ask unless you
make it clear why they are there, so I'd like to add a big explanation for those people
that will try to track it down. Costs nothing, may reduce support load here, may reduce
anxiety for devs

```suggestion

      <!-- This may generate a warning during your build:

           > property#android.adservices.AD_SERVICES_CONFIG@android:resource
           > was tagged at AndroidManifest.xml:23 to replace other declarations
           > but no other declaration present

           You may safely ignore this warning.

           We must include this in case you also use Firebase Analytics in some
           of its configurations, as it may also include this file, and the two
           will collide and cause a build error if we don't set this one to take
           priority via replacement.

           https://github.com/invertase/react-native-google-mobile-ads/issues/657

      -->
      <property
```